content/install-hook-userstylesworld.js

```diff
@@ -15,6 +16,12 @@
     && allowedOrigin === event.origin
     ) {
       sendPostMessage({type: 'usw-remove-stylus-button'});
+
+      if (location.pathname === '/api/oauth/authorize_style/new') {
```

Comment:
@Gusted, I need a way to get the style id here, which was passed to USW in `tokenMan.getToken('userstylesworld', true, style.id)`. Is it possible?

Reply:
I thought it was possible but I ended up with too much code, and the workaround was `_isUswLinked` added to the current style, and use the `find` function to get the style that way. If their is some other way around, you want to get the windowID of the opened pop-up(somehow) and have some object that pairs and windowID to the corresponding style's ID. You are encountering the same problem I had then, and my best solution still comes up for `_isUswLinked` which isn't ideally.

Reply:
I thought about passing styleID in your [OAuthStyleNewPost](https://github.com/userstyles-world/userstyles.world/blob/main/handlers/oauthProvider/authorize_style.go#L185)'s return and then somehow exposing it via postMessage maybe. Is this what you tried?

Reply:
Not really what've tried, but give me a spin with this. I might have a good idea for this one.